packages/graphql/src/modules/user/infra/mongoose/models/UserModel.ts

```diff
@@ -57,7 +57,7 @@ const ProfileImageSchema = new mongoose.Schema(
   },
 );
 
-export type IUserDocument = User & Document;
+export type IUserDocument = User.Type & Document;
```

Comment:
Why do we need a specific `Type` property? Is there some advantage in this approach? Declaring just with `User` is simpler.